docs/static/resources/openapi.json

```diff
@@ -1358,11 +1346,6 @@
             "nullable": true,
             "type": "array"
           },
-          "granularity": {
-            "description": "Name of temporal column used for time filtering. For legacy Druid datasources this defines the time grain.",
-            "nullable": true,
-            "type": "string"
-          },
```

Comment:
I still have trouble wrapping my head around these different granularities and when they're in the main payload vs extras, but shouldn't this still be here, only without the reference to legacy Druid? I looked at the `QueryObject` class, and `granularity` still seems to be featured in the constructor/property list.

Alternatively, do we need to update line 1367 which states that `granularity_sqla` "is deprecated, use `granularity` instead."?

Reply:
@villebro you're right and I've reverted the deletion. There is already a comment stating that `granularity_sqla` is deprecated.